<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite errors="0" failures="0" file="API/test_tenantRegistration.py" name="API.test_tenantRegistration.TenantRegistration-20200213070550" skipped="0" tests="5" time="0.016" timestamp="2020-02-13T07:05:50">
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="85" name="test_tenant_Email" time="0.011" timestamp="2020-02-13T07:05:50">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="35" name="test_tenant_allCorrect" time="0.002" timestamp="2020-02-13T07:05:50">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="71" name="test_tenant_noFirstName" time="0.001" timestamp="2020-02-13T07:05:50">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="57" name="test_tenant_noLastName" time="0.001" timestamp="2020-02-13T07:05:50">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="98" name="test_tenant_noPassword" time="0.001" timestamp="2020-02-13T07:05:50">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="1" file="API/test_PropertyCRUD.py" name="API.test_PropertyCRUD.UpdateProperty-20200213070550" skipped="0" tests="5" time="0.014" timestamp="2020-02-13T07:05:51">
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="297" name="test_create_property_bad_field_bath" time="0.002" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="279" name="test_create_property_bad_field_city" time="0.001" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="316" name="test_create_property_bad_field_mult" time="0.001" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="261" name="test_create_property_bad_field_street" time="0.001" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="193" name="test_update_property_allCorrect" time="0.008" timestamp="2020-02-13T07:05:51">
			<failure message="'failure' != 'success'
- failure
+ success
" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 615, in run
    testMethod()
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/test_PropertyCRUD.py", line 214, in test_update_property_allCorrect
    self.assertEqual(responseData.get(STATUS), SUCCESS)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: 'failure' != 'success'
- failure
+ success

]]>			</failure>
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="0" file="API/test_PropertyManagerRegistration.py" name="API.test_PropertyManagerRegistration.PMRegistration-20200213070550" skipped="0" tests="5" time="0.006" timestamp="2020-02-13T07:05:51">
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="37" name="test_pm_allCorrect" time="0.002" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="78" name="test_pm_missing_email" time="0.001" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="56" name="test_pm_missing_firstName" time="0.001" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="67" name="test_pm_missing_lastName" time="0.001" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="89" name="test_pm_missing_password" time="0.001" timestamp="2020-02-13T07:05:51">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="0" file="API/test_PropertyManagerRegistration.py" name="API.test_PropertyManagerRegistration.PMRegistrationRoopairs-20200213070550" skipped="0" tests="3" time="0.004" timestamp="2020-02-13T07:05:52">
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistrationRoopairs" file="API/test_PropertyManagerRegistration.py" line="158" name="test_pm_allCorrectPlusProps" time="0.003" timestamp="2020-02-13T07:05:52">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistrationRoopairs" file="API/test_PropertyManagerRegistration.py" line="139" name="test_pm_noEmail" time="0.001" timestamp="2020-02-13T07:05:52">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistrationRoopairs" file="API/test_PropertyManagerRegistration.py" line="148" name="test_pm_noPass" time="0.001" timestamp="2020-02-13T07:05:52">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="0" file="API/test_tenantLogin.py" name="API.test_tenantLogin.TenantLogin-20200213070550" skipped="0" tests="7" time="0.010" timestamp="2020-02-13T07:05:53">
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="36" name="test_tenant_allCorrect" time="0.004" timestamp="2020-02-13T07:05:53">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="63" name="test_tenant_incorrectEmail" time="0.001" timestamp="2020-02-13T07:05:53">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="87" name="test_tenant_incorrectEmailField" time="0.001" timestamp="2020-02-13T07:05:53">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="103" name="test_tenant_incorrectFields" time="0.001" timestamp="2020-02-13T07:05:53">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="71" name="test_tenant_incorrectPass" time="0.001" timestamp="2020-02-13T07:05:53">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="79" name="test_tenant_incorrectPassAndEmail" time="0.001" timestamp="2020-02-13T07:05:53">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="95" name="test_tenant_incorrectPassField" time="0.001" timestamp="2020-02-13T07:05:53">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="2" file="API/test_PropertyCRUD.py" name="API.test_PropertyCRUD.CreateProperty-20200213070550" skipped="0" tests="6" time="0.011" timestamp="2020-02-13T07:05:54">
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="150" name="test_create_property_bad_field_bath" time="0.001" timestamp="2020-02-13T07:05:54">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="134" name="test_create_property_bad_field_city" time="0.001" timestamp="2020-02-13T07:05:54">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="167" name="test_create_property_bad_field_mult" time="0.001" timestamp="2020-02-13T07:05:54">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="118" name="test_create_property_bad_field_street" time="0.001" timestamp="2020-02-13T07:05:54">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="37" name="test_create_property_allCorrect" time="0.004" timestamp="2020-02-13T07:05:54">
			<failure message="'failure' != 'success'
- failure
+ success
" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 615, in run
    testMethod()
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/test_PropertyCRUD.py", line 63, in test_create_property_allCorrect
    self.assertEqual(responseData.get(STATUS), SUCCESS)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: 'failure' != 'success'
- failure
+ success

]]>			</failure>
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="82" name="test_create_property_duplicate" time="0.003" timestamp="2020-02-13T07:05:54">
			<failure message="'failure' != 'success'
- failure
+ success
" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 615, in run
    testMethod()
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/test_PropertyCRUD.py", line 107, in test_create_property_duplicate
    self.assertEqual(responseData.get(STATUS), SUCCESS)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: 'failure' != 'success'
- failure
+ success

]]>			</failure>
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="0" file="API/test_PropertyManagerLogin.py" name="API.test_PropertyManagerLogin.PropertyManagerLogin-20200213070550" skipped="0" tests="9" time="0.011" timestamp="2020-02-13T07:05:56">
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="35" name="test_pm_allCorrect" time="0.002" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="71" name="test_pm_incorrectEmailField" time="0.001" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="87" name="test_pm_incorrectFields" time="0.001" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="79" name="test_pm_incorrectPassField" time="0.001" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="95" name="test_pm_propOnRoopairs" time="0.003" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="114" name="test_pm_propOnRoopairsXFour" time="0.002" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="47" name="test_pm_wrongEmail" time="0.001" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="55" name="test_pm_wrongPass" time="0.001" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="63" name="test_pm_wrongPassAndEmail" time="0.001" timestamp="2020-02-13T07:05:56">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
</testsuites>
